php/libraries/NDB_BVL_Battery.class.inc

```diff
@@ -312,30 +312,24 @@ class NDB_BVL_Battery
         if (!is_null($stage)) {
             $query .= ", b.instr_order as instrument_order";
         }
-        $query  .= " FROM flag AS f, test_names as t, test_subgroups as ts ";
+        $query  .= " FROM flag f
+            	JOIN test_names t ON (f.Test_name=t.Test_name)
+            	JOIN test_subgroups ts ON (ts.ID = t.Sub_group) ";
         $qparams = array('SID' => $this->sessionID);
 
         if (!is_null($stage)) {
-            $query .= ", test_battery AS b";
+            $query .= "
+            	LEFT JOIN test_battery b ON (t.Test_name=b.Test_name) ";
```

Comment:
Should this still be edited if you already put a new function above?
Other things might depend on this specific behaviour